aep/general/0124/aep.md.j2

```diff
@@ -1,5 +1,86 @@
 # Resource association
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/57) for more
-information.
+APIs sometimes have resource hierarchies that can not be cleanly expressed in
+the usual tree structure. For example, a resource may have a many-to-one
+relationship with two other resource types instead of just one. Alternatively,
+a resource may have a many-to-many relationship with another resource type.
+
+## Guidance
+
+A resource **must** have at most one canonical parent, and `List` requests
+**must not** require two distinct "parents" to work.
+
+### Multiple many-to-one associations
+
+If a resource has a many-to-one relationship with multiple resource types, it
+**must** choose at most one of them to be the canonical parent. The resource
+**may** be associated with other resources through other fields on the
+resource.
+
+{% tab proto %}
+
+{% sample 'multiple_many_to_one.proto', 'message Book' %}
+
+{% tab oas %}
+
+{% sample 'multiple_many_to_one.oas.yaml', 'Book:' %}
+
+{% endtabs %}
+
+When listing resources with multiple associations in this way, the RPC **must**
+treat the `string parent` field as required as discussed in [AEP-132][], and
```

Comment:
```suggestion
treat the `string parent` field as required as discussed in [list](/list), and
```

we're trying to move toward using the "slug" (a new concept in AEPs), rather than numbers.

Reply:
Do we care about putting links at the bottom at the bottom of the page and then using empty references in the document?